### DISCUSSION TITLE
In Progress/In Review Status Icon

### DISCUSSION THREAD

**Discussion by ceyerkes** [twilio-labs/paste#1767] [Request: Icon] [answered]
Hi! Laura Summers and I are in need of an "In Progress" or "In Review" icon for two different areas of A2P registration that we're working on.

We've been using "ProcessInProgress" for our mockups:
![Screen Shot 2021-08-12 at 3 45 16 PM](https://user-images.githubusercontent.com/87141551/129274190-d5f16d69-6587-408a-a402-bf77d8aa069d.png)

But it doesn't look like it's part of the status icon group, and is slightly smaller than the other ones we're using:
![Screen Shot 2021-08-12 at 3 46 30 PM](https://user-images.githubusercontent.com/87141551/129274307-3b42df30-2d0d-4ad1-84da-b2642b1555f9.png)

Wondering if we should bump that one up in size a bit to match the other status icons and use it? Happy to provide more context as to use case if that's helpful - open to any ideas, thanks!

**Comment by nkrantz** [comment, marked as answer]
Hi @ceyerkes! We've recently published all of our [Status Icons to Figma](https://www.figma.com/file/S4z0Kqjb9AYosnkRQldLte/Paste-Patterns?node-id=308%3A27277), so the solution here would be to swap out your Success Icons for ProcessSuccess Icons. That way, you'll be using all status icons and they'll be visually aligned. The process icons are designed to only be used with each other. Hope this helps!

**Reply by ceyerkes** [reply]
Ahhh I see - makes sense. Thanks @nkrantz!

**Reply by ceyerkes** [reply]
Hey @nkrantz after chatting with my team, I got some more clarity on this issue. We're currently using the more accessible top row version of these icons everywhere in the product, and would like to continue to do so. We'll use the neutral "i" for now for "in progress" statuses. Someone from our team will hop on office hours and give more context, since we'd like to understand when to use each set. Thanks!
